postgres-debezium-ksql-elasticsearch/postgres-debezium-ksql-elasticsearch-docker-short.adoc

```diff
@@ -1,15 +1,24 @@
-= Streaming ETL demo - Enriching event stream data with CDC data from Postgres, stream into Elasticsearch
+= Streaming ETL demo
 Robin Moffatt <robin@confluent.io>
 v1.03, 05 Jul 2018
+:revdate: 2019-03-22 11:14:37 -0600
+:revby: viktor at confluent dot io 
 
-This is designed to be run as a step-by-step demo. The `ksql_commands.sql` should match those run in this doc end-to-end and in theory you can just run the file, but I have not tested it. PRs welcome for a one-click script that just demonstrates the end-to-end running demo :)
+Enriching event stream data with CDC data from Postgres, stream into Elasticsearch
 
-c.10 minutes
+toc::[]
 
-*Slides: https://speakerdeck.com/rmoff/streaming-etl-in-practice-with-postgresql-apache-kafka-and-ksql[Streaming ETL in Practice with PostgreSQL, Apache Kafka, and KSQL]*
```

Comment:
Useful to keep in, no?

Reply:
Those are there. I just rewrote this line with author attributions

Reply:
I moved this to line 21 

https://github.com/confluentinc/examples/pull/136/files#diff-bce90ca9f1571f481ceb433189cfd281R19